Modules/AwardDKP.lua

```diff
@@ -0,0 +1,105 @@
+local _, core = ...;
+local _G = _G;
+local MonDKP = core.MonDKP;
+local L = core.L;
+
+local function GetEligibleGuildMembers(onlineOnly, sameZone, currZone)
```

Comment:
Is `onlineOnly` and `sameZone` configurable through UI?

Reply:
Yes, and it is automatically set based on those options since in the same file i do
```
local standbyEligible = GetEligibleGuildMembers(MonDKP_DB.modes.OnlineOnly, MonDKP_DB.modes.SameZoneOnly, GetRealZoneText())
```
so the list of eligible guild members is filtered based on the current options.